src/motorControl.cpp

```diff
@@ -199,9 +224,17 @@ void MotorControl::exec() {
             platform->startCooldown();
         }
     }
+    if (motorTest) {
+        // testing motor ramp
+        Serial.println("Motor testing......");
+        Serial.print("Current Speed: ");
+        Serial.println(motor->getCurrentSpeed());
+        if(motor->isTargetVelocityReached())
+            stopMotion();

```

Comment:
Perhaps instead of calling `stopMotion`, you could bypass that call, and instead set the velocity to zero in this code.   That may allow you to bypass some of the extra changes made in the `exec` check.   When you check if the velocity hits zero, reset the state variables.  Essentially bypass the various `isStopping` checks.

Reply:
Good idea - thanks